docs/source/mayavi/installation.rst

```diff
@@ -142,6 +142,39 @@ After installing VTK Python, you can install mayavi using `pip`::
 .. _PyPA: https://packaging.python.org/en/latest/current/
 .. _PyPI: https://pypi.python.org/pypi
 
+.. _install-with-edm:
+
+Installing with `Enthought Deployment Manager(EDM)`
+.....................
+
+EDM is Enthought's freely available installer. It can be downloaded from 
```

Comment:
We could mention that the advantage of this installation method over `pip`/`easy_install` is that already compiled dependencies come automatically.